base/libgit2/types.jl

```diff
@@ -274,20 +274,20 @@ immutable DiffFile
     id::Oid
     path::Cstring
     size::Int64
-    flags::Cuint
```

Comment:
It is indeed better to match the declaration in the header but this shouldn't cause any actual ABI issue on any platforms.